<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSensorThingsSubsetEditorBase</class>
 <widget class="QWidget" name="QgsSensorThingsSubsetEditorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OGC SensorThings Data Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="mEditorGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>OGC SensorThings Filter Expression</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblDataUri">
     <property name="text">
      <string>Datasource</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Fields</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item row="0" column="0">
       <widget class="QListView" name="lstFields">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;List of fields in this vector file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsCollapsibleGroupBox" name="groupBox4">
     <property name="title">
      <string>Operators</string>
     </property>
     <layout class="QGridLayout" name="_3">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item row="3" column="3">
       <widget class="QPushButton" name="mButtonNow">
        <property name="text">
         <string>now()</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="mButtonEq">
        <property name="text">
         <string>eq</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="mButtonLe">
        <property name="text">
         <string>le</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="mButtonAnd">
        <property name="text">
         <string>and</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QLabel" name="mLabelLogical_2">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="mLabelLogical">
        <property name="text">
         <string>Logical operators</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="mButtonAdd">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="mButtonGt">
        <property name="text">
         <string>gt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="mButtonGe">
        <property name="text">
         <string>ge</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QLabel" name="mLabelArithmetic">
        <property name="text">
         <string>Arithmetic</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="mButtonDiv">
        <property name="text">
         <string>div</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="mButtonLt">
        <property name="text">
         <string>lt</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="mButtonSub">
        <property name="text">
         <string>sub</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="mButtonNot">
        <property name="text">
         <string>not</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="mButtonMod">
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QLabel" name="mLabelComparisons">
        <property name="text">
         <string>Comparisons</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="mButtonMul">
        <property name="text">
         <string>mul</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="mButtonNe">
        <property name="text">
         <string>ne</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="mButtonOr">
        <property name="text">
         <string>or</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
